### ISSUE TITLE
Formatting of glossary terms

### ISSUE THREAD

**Issue by kcopas** [gbif/doc-freshwater-data-publishing-guide#11]
> should glossary terms be underlined and linked whenever they appear or only the first time they appear? I noted both ways...

Our standard is to link glossary terms the first time in each section of a document. The number of times terms are used in this document makes it extremely difficult to track. I'd welcome specific suggestions where you feel it may help clarify statements

**Comment by jenlento** [comment]
I agree that this is difficult to track. Perhaps we can see if there are any comments or any feedback on this when it goes to review.

**Comment by schmikloi** [comment]
I agree with Jen. But maybe we just stick to the plan: first time mentioned in a section. If it appears in 1.1. does it then also have to be linked in 1.1.1?
I can check this once again...

**Comment by kcopas** [comment]
The intent is to have them linked one time in each separated header section, regardless of level—§1.1, §1.1.1, §1.1.2…

Lmk if you find any missing—I will keep the issue open until one of us gives things a close look.
